### PR TITLE
Extract facilitators from slide to README on Japan

### PR DESCRIPTION
**Pull request by atoato88**
This PR extracts facilitators list from slide file to `README.md` on each Japan event.
As of now, facilitators list is included in workshop slide file.

It's good to extract informations which depends on each event to `README.md`, because that workshop slide for Japan event was moved to `japan/assets` folder as common asset.

### CODE REVIEW

**Comment by atoato88** [comment]
/assign @s-ito-ts 
Could you review PR's contents and check profile image?

File: japan/cndt-2019/README.md [hunk 1]
```diff
@@ -11,6 +11,15 @@ July 22th 2019 This is the location of our CNDT 2019 Kubernetes Upstream Trainin
...
 
+## Facilitators
+
+| Icon | Name | Affiliation | Focus on, Role.. |
+| ------------- | ------------- | ------------- | ------------- |
+|<a href="https://github.com/atoato88"><img src="https://avatars.githubusercontent.com/u/748740?s=50"></a>| <a href="https://github.com/atoato88">Akihito Inoh</a> | NEC Solution Innovators | - sig-cluster-lifecycle<br> - k/dashboard i18n ja |
+|<a href="https://github.com/shu-mutou"><img src="https://avatars.githubusercontent.com/u/12838129?s=50"></a>| <a href="https://github.com/shu-mutou">Shu Muto</a> | NEC Solution Innovators | - sig-ui<br> - Approver of k/dashboard |
+|<a href="https://github.com/k-toyoda-pi"><img src="https://avatars.githubusercontent.com/u/26761953?s=50"></a>| <a href="https://github.com/k-toyoda-pi">Kohei Toyoda</a> | NEC Solution Innovators | - sig-testing |
+|<a href="https://github.com/s-ito-ts"><img src="https://avatars.githubusercontent.com/u/42854771?s=50"></a>| <a href="https://github.com/s-ito-ts">Shingo Ito</a> | NEC Solution Innovators | - sig-testing |
+
```
Comment (shu-mutou) [inline japan/cndt-2019/README.md:22]: Add following guys also.
- mkimuram
- oomichi

Comment (atoato88) [reply japan/cndt-2019/README.md:24]: I've updated content.

Comment (shu-mutou) [reply japan/cndt-2019/README.md:24]: lgtm

**Pull request merged**